backend/clients/anthropic_client.py

```
@@ -0,0 +1,188 @@
+        if thinking:
+            params["thinking"] = {
+                "type": "enabled",
+                "budget_tokens": 16000
+            }
+
+        response = self.client.messages.create(**params)
+        assistant_response, tool_use_json = self._get_stream_response(response)
+        #self.messages.append({"role": "assistant", "content": assistant_response})
+        return assistant_response, tool_use_json
+        
+    async def send_message_async(self, message: str, queue: asyncio.Queue, stream: bool = True, tools: list[dict] = [], thinking: bool = False) -> tuple[str, dict]:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)